src/features/pageObjects/components/PageObjList.tsx

```diff
@@ -86,7 +89,14 @@ export const PageObjList: React.FC<Props> = (props) => {
 
   return (
     <div>
-      <PageObjListHeader {...{ ...props, toggleExpand, isExpanded, setActivePanel }} />
+      <PageObjListHeader
+        {...{
+          template: currentPageObject?.framework === FrameworkType.Vividus ? vividusTemplate : jdiTemplate,
```

Comment:
maybe it's better to create a separate variable for calculating props and use it in PageObjListHeader